app/script/view_model/bindings/CommonBindings.js

```diff
@@ -175,10 +176,10 @@ ko.bindingHandlers.resize = (function() {
           {
             event: {
               focus() {
-                resizeTextarea(element);
+                throttledResizeTextarea(element);
```

Comment:
Any reason why these are not single line arrow functions? `focus: () => throttledResizeTextarea(element),`

Reply:
The only reason I can think of is legacy.